docs/earthfile/earthfile.md

```diff
@@ -1598,12 +1595,37 @@ The `PIPELINE` command is in beta status and is only useful for Earthly CI.
 
 #### Description
 
-The `PIPELINE` command is used to declare that the current target is an Earthly CI pipeline. The `PIPELINE` command must be the first command in the target.
+The `PIPELINE` command is used to declare that the current target is an Earthly CI pipeline. The `PIPELINE` command must be the first command in the target. The `PROJECT` declaration in the Earthfile is also required, and it must match the project the repository is added to, in order for the pipeline to be executed.
```

Comment:
I struggle to understand `and it must match the project the repository is added to, in order for the pipeline to be executed`.

Maybe breaking it up into a separate paragraph, and rewording it to:

```suggestion
The `PIPELINE` command is used to declare that the current target is an Earthly CI pipeline. The `PIPELINE` command must be the first command in the target.

To use a `PIPELINE`, you must also declare a `PROJECT` in the Earthfile.
This `PROJECT` must match the name of the CI's project that references the git repository
```

would make it easier?